backend/src/routes/cars.routes.ts

```diff
@@ -1,10 +1,11 @@
 import { Router } from "express";
-import { deleteCarController, updateCarController } from "../controllers/cars.controller";
+import { deleteCarController, getByIdCarsController, updateCarController } from "../controllers/cars.controller";
 
 const carRoutes = Router()
 
 carRoutes.post("",)
 carRoutes.get("",)
+carRoutes.get(":id",getByIdCarsController)
```

Comment:
Some endpoints have "/" and others don't. Can you review it?

Reply:
I've corrected the inconsistency mentioned. Thank you